Docs cli and server (#3479)
RasaHQ/rasa: docs/command-line-interface.rst

Command                    Effect
=========================  ===================================================================================
``rasa init``              Creates a new project, with example training data, actions, and config files.
``rasa run``               Starts a server with your model loaded. See the :ref:`_section_http` docs for details.
``rasa run actions``       Starts an action server using the Rasa SDK.
``rasa shell``             Loads your trained model and lets you talk to your assistant on the command line.
``rasa train``             Trains a model using your NLU data and stories, saves trained model in ``./models``.
``rasa interactive``       Starts an interactive learning session to create new training data by chatting.
``rasa test``              Tests a trained model using your test NLU data and stories.

> ``rasa visualize``         Visualize Rasa Core stories.

Reviewer: ```suggestion
``rasa visualize``         Visualize stories.
```

                        directory. (default: models)

> In case you start the chat session only with a trained NLU model, ``rasa shell`` allows

Reviewer: ```suggestion
In case you start the shell with an NLU-only model, ``rasa shell`` allows
```
